tests/fv/fv_test.go

```diff
@@ -406,6 +406,62 @@ var _ = Describe("kube-controllers FV tests", func() {
 		})
 	})
 
+	Context("ServiceAccount Profile FV tests", func() {
+		var profName string
+		BeforeEach(func() {
+			saName := "peanutbutter"
+			nsName := "default"
+			profName = "ksa." + nsName + "." + saName
+			sa := &v1.ServiceAccount{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: saName,
```

Comment:
nit: I'd explicitly set the `Namespace: nsName` here for clarity.

Reply:
yes.